gtk4/examples/content_provider.rs

```diff
@@ -0,0 +1,95 @@
+use gtk::prelude::*;
```

Comment:
The example needs an entry in `examples/Cargo.toml` otherwise it won't be tested as part of the CI, along with a README.md explaining very briefly what the example is about. See how it's handled for the other examples

Reply:
I didn't put in examples/ but in gtk4/examples. Why wouldn't it be built? I think it's not generally useful, and it's more of a test actually, but it requires a display connection.

Reply:
Well, it should be moved to examples instead of gtk4/examples so it's not shipped with the crate. The examples are used as a build test for the API/bindings to ensure things are kept working.

Reply:
ok, done